website/docs/contributing/writing-guide.md

```diff
@@ -77,6 +77,46 @@ Rather than use relative paths, we strongly suggest the following approach, sinc
 
 `![FerretDB logo](/img/logo-dark.png)`.
 
+## Tags
+
+Tags are an important part of every blog post, and they appear at the top of the front matter.
+They make it easy for readers to search and identify related blog posts based on their categories or subject matter.
+You can view all [currently listed tags here](https://blog.ferretdb.io/tags/).
+
+Please note that tags are case-sensitive, such that `Release` and `release` are two separate tags.
+Unless distinctly written (as in the case with CI/CD, DevOps), all tags should be in small-case.
```

Comment:
Well, I would say "MongoDB" and "GUI" is in the same group as "DevOps", yet we say "mongodb gui" below. Let's just remove that "unless" part